apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx

```diff
@@ -206,6 +206,12 @@ export default async function Details({params}: Props) {
             >
               {object.description}
             </Metadata>
+            <Metadata
+              translationKey="locationCreated"
+              enrichmentType={AdditionalType.LocationCreated}
+            >
+              {object.locationCreated?.name}
```

Comment:
This show the actual location, right? For consideration: we could also show the country of which it is a part (`isPartOf`), depending on the UI wishes (e.g. `Jakarta (Indonesia)`).

Reply:
Good point, I have added `isPartOf`